medical-portal/src/UI/src/app/get-help/get-help.component.html

```diff
@@ -27,7 +27,12 @@ <h1>Getting help with Driver Medical Fitness</h1>
               the DMER Case ID is located on the letter.
             </p>
             <div id="u84" class="ax_default image">
-              <img id="u84_img" class="img" src="assets/images/u84.svg" />
+              <img
```

Comment:
is this prettier making things uglier again? Alt is required for accessibility validation

Reply:
Yes Alt is required for the accessibility validation. There were three changes that came back so it was quick to fix.